Feature/improve UI

ZagreDo opened the pull request:
Added some new pages, improved old ones with more functionality and nicer esthetics.

ZagreDo committed:
Leaderboard is cleaner.

ZagreDo committed:
Minor Changes

ZagreDo committed:
More aesthetic interface

ZagreDo committed:
Minor fixes

ZagreDo committed:
Added some filtering in leaderboard.

Rytisok commented on Encounter Me/Pages/Leaderboard.razor:
```diff
@@ -1,43 +1,100 @@
+                |
+                <a href="@($"Leaderboard/Factions")">Factions</a>
+            </div> 
+            <div style="margin-right:5%">
+                <select style="width:50px;float:right" @onclick="@(() => doUsers5(pageNumber))" @bind=@pageNumber>
+                    @for (int p = 1; p <= (Users.Count() / 5); p++)
+                    {
+                        <option value="@p">@p</option>
+                    }
+                    @if (Users.Count() % 5 != 0)
+                    {
+                        <option value=@((Users.Count() / 5)+1)>@((Users.Count() / 5)+1)</option>
+                    }
```
What is this drop down for?

ZagreDo replied:
It generates, number of pages to check out all leaaderboard.

Rytisok committed:
Merge branch 'master' into feature/ImproveUI

Rytisok committed:
deleted outdated migration

Rytisok committed:
Leaderboard restyle

Leaderboard restyled for new style

Rytisok committed:
profile edit form fields restricted

profile edit form fields restricted

Commit:
Add default value to select

Commit:
Merge branch 'feature/ImproveUI' of https://github.com/Rytisok/EncounterMe into feature/ImproveUI

Commit:
Merge branch 'feature/ImproveUI' of https://github.com/Rytisok/EncounterMe into feature/ImproveUI

Rytisok committed:
user image shown on trail complete

Rytisok committed:
Visual fixes

-navbar now collapses
-registration form is now scrollable

Rytisok committed:
Map search button improvements

Rytisok committed:
font update and leaderboard page rework sketched

Rytisok committed:
bars representing each factions control added

-bars representing each factions control added
-api updated to fetch each factions control percentage
-leaderboard page renamed to statistics

Rytisok committed:
profile paged style improved
